config/0chain_blobber.yaml

```diff
@@ -99,7 +99,7 @@ challenge_response:
   max_retries: 20
 
 healthcheck:
-  frequency: 60s # send healthcheck to miners every 60 seconds
+  frequency: 60m # send healthcheck to miners every 60 seconds
```

Comment:
please also change the comment. 

`# send healthcheck to miners every 1 hour`

Reply:
updated